src/datasources/entity-ds/mapping/FilterMapping.js

```diff
@@ -22,13 +22,25 @@ export class FilterMapping {
         const self = this;
         const filter = new API.Filter();
         filter.limit = 0;
+        filter.orderBy = [];
 
-        _.each(uiFilter.query.clauses, function(eachClause) {
+        uiFilter?.query?.clauses?.forEach((eachClause) => {
```

Comment:
What is this madness?

Reply:
syntactic sugar for `if (uiFilter && uiFilter.query && uiFilter.query.clauses)`, which I am tired of typing.  Proposal for a future ES spec, it's in the candidate stage.  Babel handles it for us, but if you reaaaaly hate it I can put it back to the old way.

Reply:
My goal is to get rid of lodash as much as possible, since it doesn't tree-shake well, so we end up with 500k of junk that minify doesn't want to mess with.

Reply:
Neat. Just hadn't see it before, tis all.